.ci.yaml

```diff
@@ -12,6 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+  linux_web:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "chrome_and_driver", "version": "version:114.0"}
+        ]
+      os: Ubuntu
```

Comment:
Is there a preference on `Ubuntu` vs `Linux` for non-desktop tests?

Reply:
Now everything is Ubuntu, and let's keep `Ubuntu` for any linux targets.